smartsim/_core/utils/helpers.py

```diff
@@ -183,12 +190,12 @@ def cat_arg_and_value(arg_name: str, value: str) -> str:
 
     if arg_name.startswith("--"):
         return "=".join((arg_name, str(value)))
```

Comment:
f"{arg_name}={value}"

Reply:
Similar for the following clauses. Using join() seems unnecessarily heavyweight here